### DISCUSSION TITLE
npm not installing in NVM version 1.1.11

### DISCUSSION THREAD

**Discussion by BhagvanReddy1** [coreybutler/nvm-windows#1137] [General]
Hi Team,

I am using **NVM 1.1.11** version & when i run **NVM install 10.15.0** encountering with below issue

 Downloading npm version 6.4.1... Error while downloading https://github.com/npm/cli/archive/v6.4.1.zip - Get "https://github.com/npm/cli/archive/v6.4.1.zip": dial tcp 20.207.73.82:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
Could not download npm for node v10.15.0.
Please visit https://github.com/npm/cli/releases/tag/v6.4.1 to download npm.
It should be extracted to C:\Install\nvm\v10.15.0
![image](https://github.com/coreybutler/nvm-windows/assets/169364369/ccccf794-67b0-4f8b-a8b4-07c0d209a472)

Could someone help here?.

**Comment by coreybutler** [comment]
First, the "team" is just me :-)

This error occurs when there is a network connectivity issue. This is most commonly encountered when a corporate proxy blocks or interferes with access to GitHub. It's worth noting that the older versions of npm are in their archive (as the URLs suggest), but Github usually redirects these requests. So, if you are behind  proxy that doesn't allow redirects, it can prevent the download from occurring. This is why the error message instructs you to download npm manually and extract it to the correct location.

The bottom line is you should check with your network admin to make sure Github redirect links are followed. It's very common for this to be allowed in browsers but not for other programs. It is not something NVM4W has control over (which is I tried to put an instructive error message).
